apps/web/app/(org)/dashboard/caps/Caps.tsx

```
@@ -190,85 +195,78 @@ export const Caps = ({
+      return yield* fiber.await.pipe(Effect.flatten);
+    }),
+    onSuccess: Effect.fn(function* () {
       setSelectedCaps([]);
-      refresh();
-    } catch (error) {
-    } finally {
-      setIsDeleting(false);
-    }
-  };
+      router.refresh();
+    }),
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)